query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/simple/multi_field_unique.rs

```diff
@@ -107,7 +107,7 @@ mod multi_field_unique {
             }
             "# },
             2012,
-            "Missing a required value at `Query.findUniqueUser.where.UserWhereUniqueInput.FirstName_LastName.UserFirstNameLastNameCompoundUniqueInput.LastName"
+            "Missing a required value at `Query.findUniqueUser.where.UserWhereUniqueInput.FirstName_LastName.UserFirstName_LastNameCompoundUniqueInput.LastName"
```

Comment:
This type name change is breaking. Is this a mistake or did you intend to change the name?